app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
     permissionChanges: {},
     bibliographicChanges: {},
     valid: Ember.computed.alias('newContributorId'),
+    permissionToggle: false,
+    bibliographicToggle: false,
+    removalToggle: false,
+    stillAdmin: Ember.computed('isAdmin', function() {
+        return this.get('isAdmin');
+    }),
+    addState: 'emptyView', // There are 3 view states on left side of Authors panel. Default state just shows search bar.
```

Comment:
Yes nested routes are worth looking into.  I will put this on the backburner for now though.